uncover/agent/hunter/request.go

```diff
@@ -0,0 +1,13 @@
+package hunter
+
+type Request struct {
+	USERNAME   string `json:"username"`
```

Comment:
Why are the `USERNAME` and `APIKEY` fields all uppercase?

Reply:
Sorry, this is a mistake of mine.